pkg/api/server/v1alpha2/plugin_logs.go

```diff
@@ -105,25 +163,41 @@ func (s *LogPluginServer) getLokiLogs(writer *logs.BufferedLog, parent, id strin
 		return status.Error(codes.Internal, "Error streaming log")
 	}
 
+	if resp.StatusCode != http.StatusOK {
+		s.logger.Errorf("Loki API request failed with status code: %d", resp.StatusCode)
```

Comment:
```suggestion
		s.logger.Errorf("Loki API request failed with HTTP status code: %d", resp.StatusCode)
```

Reply:
This is the differentiate between the HTTP request error and the Loki response error below.